torchaudio/csrc/ffmpeg/stream_reader/buffer/common.cpp

```diff
@@ -96,6 +96,16 @@ torch::Tensor get_buffer(
   return torch::empty(shape, options);
 }
 
+torch::Tensor get_buffer16(
```

Comment:
I think this function can be merged to `get_buffer`, by adding `dtype` argument which defaults to `kUInt8`.